src/main/java/wooteco/subway/line/LineService.java

```diff
@@ -14,22 +14,20 @@ private LineService(LineDao lineDao) {
         this.lineDao = lineDao;
     }
 
-    public Line add(String name, String color) {
-        List<Line> lines = lineDao.findAll();
-        Line line = new Line(name, color);
-        validateDuplicatedLine(lines, line);
+    public Line add(LineRequest lineRequest) {
+        validateDuplication(lineRequest.getName());
+        Line line = new Line(lineRequest.getName(), lineRequest.getColor());
```

Comment:
> 2. DTO에서 값을 꺼내서 도메인 객체를 만들지 말고, DTO에서 바로 도메인 객체를 생성할 수 있는 책임을 가지도록 해본다.

이 부분에 대한 적용이 되어있지 않은 것 같아요 ㅎㅎ
```suggestion
        Line line = lineRequest.toLine();
```
이런 식의 적용을 이야기한 거였어요.
이미 Request DTO에서 Line 생성에 필요한 모든 정보를 갖고 있으니까요 :)